loops.js

```diff
@@ -1,42 +1,49 @@
 // ==========================================
-// Opdracht 1. Maak een for-loop die 3 keer "Hoera!" in de terminal logt
+// Opdracht 1. Maak een for loop die het volgende patroon in de terminal print:
 // Verwachte uitkomsten:
-// Hoera!
-// Hoera!
-// Hoera!
-// ==========================================
+// *
+// **
+// ***
+// ****
+// *****
 
+// Tip: je kunt de .repeat() methode gebruiken om een karakter een n aantal keer te herhalen...
+// Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
+// ==========================================
+// Opdracht 1. Maak een for loop die het volgende patroon in de terminal print:
+// Verwachte uitkomsten:
+// *
+// **
+// ***
+// ****
+// *****
 
+let line = "";
+for (let i = 0; i < 5; i++) {
+    line += "*";
+    console.log(line);
+}
 
 // ==========================================
-// Opdracht 2. Maak een for-loop die 4 keer het woord "loop..." logt, en bij de vijfde loop het woord "klaar!"
+// Opdracht 2. Maak een for-loop die 4 keer het woord "loop..." logt, en bij de laatste (vijfde) loop het woord "klaar!"
 // Verwachte uitkomsten:
 // loop...
 // loop...
 // loop...
 // loop...
 // klaar!
- // ==========================================
-
-
-
-// ==========================================
-// Opdracht 3. Maak een for-loop die van 0 tot 15 loopt. Voor ieder getal wordt in de console gelogd of het getal even of oneven is.
-// Verwachte uitkomsten:
-// 0 is even
-// 1 is oneven
-// 2 is even
-// 3 is oneven
-// .... etc.
-// 15 is oneven
-
-// Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over. Google daarom eens naar 'mdn webdocs remainder operator'.
 // ==========================================
 
-
+for (let i = 0; i < 5; i++) {
```

Comment:
if statement is overbodig. 
na `for  (let i = 0; i < 4; i++){ log "loop..."}` gewoon klaar loggen.

Reply:
Ik ben er zelf ook heel goed in: moeilijk denken....